src/AppBundle/Form/Type/EditorType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+
+/**
+ * Defines a custom form field type used to display a markdown editor with preview functionality.
+ *
+ * See http://symfony.com/doc/current/cookbook/form/create_custom_field_type.html
+ *
+ * @author Nikolai Plath <der.el.kuku@gmail.com>
+ */
+class EditorType extends AbstractType
```

Comment:
If only one value here is to override the block name used to render the form type then, use the `block_name` option instead (bonus plus to this PR) and remove entirely this form type.

For example:
```php
// AppBundle\Form\PostType

->add('content', null, [
    'block_name' => 'editor',
])
```
```twig
{# form/fields.html.twig #}

{% block _post_editor_widget %}
    ...
    {{ block('textarea_widget') }}
    ...
{% endblock %}
```
http://symfony.com/doc/current/reference/forms/types/form.html#block-name
http://symfony.com/doc/current/form/form_customization.html#form-customization-sidebar
http://symfony.com/doc/current/form/form_themes.html#form-template-blocks